js/articleView.js

```diff
@@ -16,7 +27,7 @@ articleView.handleMainNav = function() {
     }
```

Comment:
There is a way to reduce the if/else statements above into a single line of code. Let's discuss, if you wish.